manifest-reference.md

```diff
@@ -479,3 +481,24 @@ By default only executables are installed, library projects can set the *library
 [install]
 library = true
 ```
+
+
+## Additional free data field
+
+Third-party tools can store their configuration inside the *extra* section.
+This section will never be evaluated by fpm itself, the only constraint imposed it that is has to be valid TOML.
+
+Since the format of this section is free, only recommendations are provided here for adding data to the *extra* section.
+
+1. Only use subtables, never add configuration data to the toplevel of the *extra* section.
+   Reasoning: different tools can avoid collisions of keynames by placing their data in separate subtables
+2. Use the concrete name of the tool rather than a generic name for the subtable.
+   Reasoning: different formatter or linter might use conflicting keywords in a *format* or *lint* subtable.
+   Also, users can tell from the table name which tool is preferred for usage with the project.
+3. Fpm plugins should use a subtable with their plugin name in the *extra.fpm* section to store their data.
```

Comment:
```suggestion
1. Only use subtables, never add configuration data to the top level of the *extra* section.
   Reasoning: different tools can avoid collisions of key names by placing their data in separate subtables.
2. Use the concrete name of the tool rather than a generic name for the subtable.
   Reasoning: different formatter or linter tools might use conflicting keywords in a *format* or *lint* subtable.
   Also, users can tell from the table name which tool is preferred to use with the project.
3. Fpm plugins should use a subtable with their plugin name in the *extra.fpm* section to store their data.
```